assets/js/main.js

```diff
@@ -52,3 +52,20 @@ $("#btn-modal-small").click(function() {
 $(".modal .close").click(function() {
   $(".modal").fadeOut();
 });
+
+
+// script to show and hide password
+$(".show-hide").click(function() {
+  $(this).toggleClass("ms-icon icon-hide");
+  var input = $($(this).attr("toggle"));
+
+  if (input.attr("type") == "password") {
+    input.attr("type", "text");
+    $('span.content-eye').text('Ocultar');
+    $(this).removeClass("ms-icon icon-view").addClass("ms-icon icon-hide");
+  } else {
+    input.attr("type", "password");
+    $('span.content-eye').text('Mostrar');
```

Comment:
could you read this text literals form the DOM in order to allow us to translate it?